### ISSUE TITLE
train my own dataset and the effect is bad

### ISSUE THREAD

**Issue by thinkyang88** [xingyizhou/CenterNet#274]
I trained my own dataset ，the train loss is 1.3 in 2th epoch and then not decreased.The val loss is 10.6 and I uesd ctdet --demo to test one picture, the  effect is very bad,detect nothing,can you help me?thanks

**Comment by xingyizhou** [comment]
Can you visualize your training data by `--debug 2` when runing `python main.py xxx`? Also make sure you haven't seen any `drop parameters xxx` or `no parameters xxx` when loading your trained model.

**Comment by thinkyang88** [comment]
I visualize your training data by --debug 2 and find that the gt heatm ap
is almost nothing,but the label information is correct.The pictures are
gt-hm,gr-out,gt-pred.The true objecrs are tennies-court.
[image: gt_hm_screenshot_25.08.2019.png]
[image: out_gt_screenshot_25.08.2019.png]
[image: out_pred_screenshot_25.08.2019.png]


Xingyi Zhou <notifications@github.com> 于2019年8月22日周四 上午2:59写道：

**Comment by xingyizhou** [comment]
Then it is most likely your training data is not in the correct format. Please make sure the bounding box is in `(x0, y0, w, h)`, not `(x0, y0, x1, y1)`.

**Comment by leavewave** [comment]
> Can you visualize your training data by `--debug 2` when runing `python main.py xxx`? Also make sure you haven't seen any `drop parameters xxx` or `no parameters xxx` when loading your trained model.

I meet the problem you described,how i can solve it? thanks ,author

**Comment by kchaitanyabandi** [comment]
> I trained my own dataset ，the train loss is 1.3 in 2th epoch and then not decreased.The val loss is 10.6 and I uesd ctdet --demo to test one picture, the effect is very bad,detect nothing,can you help me?thanks

Hi @thinkyang88 : Could you please let us know what changes you made to the code to train on your own dataset?
